_posts/2021-02-20-kotlin-use-site-targets.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Unwrapping Use-site Targets in Kotlin"
+date:   2021-03-02 10:00:00 +0200
+comments: true
+img:
+  href: 2021-03-02-dogs-of-berlin.png
+  copyright: Dogs of Berlin by Christian Alvart
+  alt: Dogs of Berlin
+---
+I like Kotlin for its expressiveness. Its interoperability with Java
+allows to easily use the existing Java ecosystem while enjoying a modern
```

Comment:
"allows **the user** to easily use the existing Java ecosystem ...

Reply:
It is more like a developer, not a user. If I replaced "the user" with "us" would it sound better?

Reply:
Us is tricky with informal texts. Could you say, "allows the developer to easily use the existing Java ecosystem..."